models/conversations.go

```diff
@@ -0,0 +1,40 @@
+package models
+
+import (
+	"context"
+)
+
+type Conversation struct {
+	ID      string `json:"id" bson:"_id,omitempty"`
+	GroupID string `json:"groupId" bson:"group_id,omitempty"`
+	Title   string `json:"title" bson:"title,omitempty"`
+}
+
+type ConversationInfo struct {
+	Title   string `json:"title" bson:"title,omitempty"`
+	GroupID string `json:"groupId" bson:"group_id,omitempty"`
+}
+
+type ConversationTitle struct {
+	Title string `json:"title" bson:"title,omitempty"`
+}
+
+type OneConversationFilter struct {
+	ID string `json:"id" bson:"_id,omitempty"`
+}
+
+type AllConversationsFilter struct {
+	ID string `json:"groupId" bson:"group_id,omitempty"`
```

Comment:
This should be called `GroupID`. You can filter conversations by group id not by ID. Also should be `json:"group_id"`